.github/workflows/functional.yml

```diff
@@ -105,9 +105,6 @@ jobs:
         env:
           RSTUF_BROKER_SERVER: amqp://guest:guest@rabbitmq:5672
           RSTUF_REDIS_SERVER: redis://redis
-          RSTUF_AUTH: "true"
-          SECRETS_RSTUF_TOKEN_KEY: test-token-key
-          SECRETS_RSTUF_ADMIN_PASSWORD: secret
```

Comment:
Are these env variables to be removed also from https://github.com/repository-service-tuf/repository-service-tuf-api/blob/main/docker-compose.yml#L93-L95?

Reply:
I don't think so

The env variables are for the development environment (`make run-dev`). We want while developing RSTUF API to use the built-in auth to make sure we are not breaking things.